frontend/src/pages/SubProjects/SubProjectContainer.js

```diff
@@ -95,8 +94,7 @@ const mapDispatchToProps = (dispatch, ownProps) => {
     liveUpdate: projectId => dispatch(liveUpdateProject(projectId)),
     showSubprojectDialog: () => dispatch(showSubprojectDialog()),
 
-    openHistory: (projectId, offset, limit) => {
-      dispatch(fetchFirstProjectHistory(projectId, offset, limit, true));
+    openHistory: () => {
```

Comment:
```suggestion
    showHistory: () => {
```

Reply:
renamed the action to `openHistory` since `showHistory` already exists as state variable